src/app/globals.css

```diff
@@ -6,19 +8,17 @@
 
 :root {
   /* Colors */
-  --color-primary: #68fe9c;
-  --color-secondary: #e2e2e2;
```

Comment:
`--color-secondary` is used in `AppHeader`. Maybe it's replaced by `--color-font`? Usually, you would name it `--color-text` instead of `--color-font`.